packages/filebrowser/src/listing.ts

```diff
@@ -2297,7 +2368,7 @@ export namespace DirListing {
       const trans = translator.load('jupyterlab');
       const name = this.createHeaderItemNode(trans.__('Name'));
       const narrow = document.createElement('div');
-      const modified = this.createHeaderItemNode(trans.__('Last Modified'));
+      const modified = this.createHeaderItemNode(trans.__('Modified'));
```

Comment:
Would it be possible to keep the original `Last Modified` here?

Since the file browser is also used in Notebook, and there is plenty of space to display it.

Reply:
Noticed in https://github.com/jupyter/notebook/pull/7333